ffs/scheduler/internal/jstore/jstore.go

```diff
@@ -36,11 +40,15 @@ type watcher struct {
 }
 
 // New returns a new JobStore backed by the Datastore.
-func New(ds datastore.Datastore) *Store {
-	return &Store{
-		ds:         ds,
-		inProgress: make(map[cid.Cid]struct{}),
+func New(ds datastore.Datastore) (*Store, error) {
+	s := &Store{
+		ds:            ds,
+		executingCids: make(map[cid.Cid]ffs.JobID),
 	}
+	if err := s.loadExecutingJobs(); err != nil {
```

Comment:
When the `JobStore` of the `Scheduler` is created, it recognizes which Jobs appear to be in "Executing" status... which is necessary to recover them and continue them.